src/components/PhoneInput/index.tsx

```diff
@@ -45,11 +43,10 @@ const StyledLabel = styled.label<PhoneInputLabelProps>`
   }
 
   &:focus-within {
-    border: 1px solid
-      ${({ theme: { colorsDeprecated } }) => colorsDeprecated.primary};
+    border: 1px solid ${({ theme }) => theme.colors.primary.borderStrong};
```

Comment:
Here too you can make it weak to follow context

```suggestion
    border: 1px solid ${({ theme }) => theme.colors.primary.borderWeak};
```

Reply:
done